### PR TITLE
Update page in AMPLIFY Unified Catalog “_index”

### PR DESCRIPTION
**Pull request by jnamburi**
Automatically generated by Netlify CMS

### CODE REVIEW

File: content/en/docs/catalog/_index.md [hunk 1]
```diff
@@ -2,12 +2,49 @@
...
 weight: 150
-date: 2020-04-28
-description: Normalize discovery for APIs from multiple gateways, classify your services to support multiple audiences (partners, IT, business), control consumer subscription to access your APIs, and extend your APIs to be reused in other integration ﬂows.
+date: 2020-04-28T00:00:00.000Z
+description: "Normalize discovery for APIs from multiple gateways, classify your
+  services to support multiple audiences (partners, IT, business), control
+  consumer subscription to access your APIs, and extend your APIs to be reused
+  in other integration flows."
 ---
+AMPLIFY Unified Catalog is a SaaS based service that is part of the AMPLIFY platform management layer, AMPLIFY Central.
```
Comment (andreamussap) [inline content/en/docs/catalog/_index.md:11]: @georgiana-caras please confirm that this is the correct name, **AMPLIFY Central** ??

File: content/en/docs/catalog/_index.md [hunk 2]
```diff
@@ -2,12 +2,49 @@
...
 
-AMPLIFY Catalog allows developers to discover and understand the endpoints and protocols that they can use to integrate quickly with your services. It simplifies access and increases the speed of building, integrating, and deploying new services and apps. Platform administrators can limit content access to specific users or teams and manage subscriptions.
+![Automate the population of the Unified Catalog from multiple source](/Images/catalog/automate-the-population-of-the-unified-catalog-from-multiple-sources.png "Automate the population of the Unified Catalog from multiple source")
+
+## Unified Catalog capabilities
+
+The Unified Catalog capabilities include:
+
+* Event-driven subscriptions management: You can create custom subscription flows for each registered environment enabling complex approval flows and integration with existing systems to streamline experience and reduce time for approval.
+* Customized documentation for each catalog asset. The documentation is written using Markdown.
```
Comment (andreamussap) [inline content/en/docs/catalog/_index.md:24]: @jnamburi is there a particular reason for you to call the Overview field/description of an asset '**documentation**'? Why not call it simply an overview of your assent?

**Comment by andreamussap** [comment]
Direct preview link for the page - https://deploy-preview-825--axway-open-docs.netlify.app/docs/catalog/

File: content/en/docs/catalog/_index.md [hunk 3]
```diff
@@ -2,12 +2,49 @@
...
+date: 2020-04-28T00:00:00.000Z
+description: "Normalize discovery for APIs from multiple gateways, classify your
+  services to support multiple audiences (partners, IT, business), control
+  consumer subscription to access your APIs, and extend your APIs to be reused
+  in other integration flows."
 ---
+AMPLIFY Unified Catalog is a SaaS based service that is part of the AMPLIFY platform management layer, AMPLIFY Central.
 
-The AMPLIFY Central and Catalog work together to help customers to have a common place to import their APIs from all of their distributed API gateways throughout their enterprise.
+AMPLIFY Central and AMPLIFY Unified Catalog work together to provide a common place for all integration assets from all of your distributed gateways through the enterprise. AMPLIFY Unified Catalog enables sophisticated integration ﬂows combining cloud and public on-premise APIs together. Alongside API interfaces, you can find Managed File Transfer (MFT) flows that move data across your enterprise and outside its boundaries.
```
[the remark this answers is not in the document]
Comment (georgiana-caras) [reply content/en/docs/catalog/_index.md:13]: They are integration assets - assets that provide integrations to different systems.

**Comment by andreamussap** [comment]
**Blocked** - Waiting for feedback!

File: content/en/docs/catalog/_index.md (hunk 1 above)
Comment (georgiana-caras) [reply content/en/docs/catalog/_index.md:11]: Confirmed.

File: content/en/docs/catalog/_index.md [hunk 4]
```diff
@@ -2,12 +2,49 @@
...
 
-The AMPLIFY Catalog provides a common view of all endpoints belonging to an organization. It enables more sophisticated integration ﬂows combining cloud and public on-premise APIs together. Alongside API interfaces, you can find Managed File Transfer (MFT) and B2B connectors that move data across your enterprise and outside its boundaries. The endpoints in the catalog can be in different environments such as in the cloud, on-premise, within a microservice mesh, or at the edge (DMZ) of the organization.
+The endpoints in the catalog are hosted in different environments, such as in the cloud, on-premise, within a microservice mesh, or at the edge (DMZ) of the organization. AMPLIFY Central manages multiple gateway environments that feed the Unified Catalog service. These gateways can come from Axway (for example, API Manager) or another vendor (for example, AWS or Azure), or they can also be repositories such as GitHub and SwaggerHub.
```
[the remark this answers is not in the document]
Comment (georgiana-caras) [reply content/en/docs/catalog/_index.md:15]: Maybe: These gateways can "be " Axway gateways (for example..) or from another vendor ...

File: content/en/docs/catalog/_index.md [hunk 5]
```diff
@@ -2,12 +2,49 @@
...
 
-AMPLIFY Catalog allows developers to discover and understand the endpoints and protocols that they can use to integrate quickly with your services. It simplifies access and increases the speed of building, integrating, and deploying new services and apps. Platform administrators can limit content access to specific users or teams and manage subscriptions.
+![Automate the population of the Unified Catalog from multiple source](/Images/catalog/automate-the-population-of-the-unified-catalog-from-multiple-sources.png "Automate the population of the Unified Catalog from multiple source")
+
+## Unified Catalog capabilities
```
[the remark this answers is not in the document]
Comment (georgiana-caras) [reply content/en/docs/catalog/_index.md:19]: Agreed

File: content/en/docs/catalog/_index.md (hunk 2 above)
Comment (georgiana-caras) [reply content/en/docs/catalog/_index.md:24]: It is okay to call it documentation, at this point we don;t want to be too specific about the supported features (the markdown documentation will be added in the Overview tab of the catalog description).

**Pull request merged**